packages/roosterjs-editor-core/lib/corePlugins/LifecyclePlugin.ts

```diff
@@ -91,10 +91,10 @@ export default class LifecyclePlugin implements PluginWithState<LifecyclePluginS
 
         this.state = {
             customData: {},
-            defaultFormat: options.defaultFormat || null,
+            defaultFormat: options.defaultFormat ?? null,
             isDarkMode: !!options.inDarkMode,
-            getDarkColor: options.getDarkColor || ((color: string) => color),
-            onExternalContentTransform: options.onExternalContentTransform,
+            getDarkColor: options.getDarkColor ?? ((color: string | null) => color ?? '#000'),
```

Comment:
why use #000 as default value?

Reply:
@ianeli1  can you provide some update for this comment? Thanks.

Reply:
Not sure what the solution is here
If given no color (`null`), it previously returned `null`, which doesn't match the typings given
Should this behavior be kept and return `string | null` instead, or should it return `""` as default?
I can only see this method being used here, which removes the styling attribute if `null` is returned
https://github.com/microsoft/roosterjs/blob/457a8e4cdad3c17d861ff2f4b5da969d892b6d39/packages/roosterjs-editor-core/lib/coreApi/transformColor.ts#L63
https://github.com/microsoft/roosterjs/blob/457a8e4cdad3c17d861ff2f4b5da969d892b6d39/packages/roosterjs-editor-core/lib/coreApi/transformColor.ts#L131-L141

Reply:
@ianeli1

Reply:
Do you mean having the definition only accept strings
`(color: string) => string`
or allowing it to return null?
`(color: string | null) => string | null`
We can do either I believe

Reply:
I prefer the first one